.github/workflows/dep.yml

```diff
@@ -25,6 +25,7 @@ on:
     paths:
       # dependency check happens only pom changes
       - '**/pom.xml'
+      - '.github/workflows/dep.yml'
```

Comment:
Is this necessary?

Reply:
`dep.yml` is designed to check and verify dependencies although doesn't bring changes in dependencies, I think it's good to leave it in paths. As for this PR, this is add to trigger the Dependency Check job.